src/migration.types.ts

```diff
@@ -85,13 +85,13 @@ export interface ErrorReport {
  * A function that can be called to migrate a dashboard from one version to another.
  * It is called within {@link produce | https://immerjs.github.io/immer/produce}, so it can safely mutate its input `dashboardState`.
  */
-export type MigrateDashboardCallback<FromDashboardState, ToDashboardState> = (
+export type MigrateDashboardCallback<
+  FromDashboardState,
+  ToDashboardState,
+  Options extends Record<string, unknown> = Record<string, unknown>,
```

Comment:
Is `unkown` better than `any`?

Reply:
Yes, `Record<string, unknown>` is recommended type to represent "any object"